### ISSUE TITLE
cannot export file if drive is already unlocked

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-export#12]
## Description

When the drive is already unlocked, the return code is `USB_BAD_PASSPHRASE` and the file is not copied to the usb drive.

## Steps to Repro

1. From the Qubes menu in the upper-lefthand corner, hover over `sd-export-usb` and select `sd-export-usb: Files`
2. Select your thumb drive folder, enter in your luks passphrase with the default selection to keep the device unlocked 
3. run `qvm-open-in-vm sd-export-usb archive.sd-export` from `sd-svs` and see that you get back  `USB_BAD_PASSPHRASE`

**Comment by creviera** [comment]
## Another way to Repro

1. From the Qubes menu in the upper-lefthand corner, hover over `sd-export-usb` and select `sd-export-usb: Files`
2. Select your thumb drive folder, enter in your luks passphrase with the `Forget password immediately` option selected
3. Run `qvm-open-in-vm sd-export-usb archive.sd-export` from `sd-svs`

**Expected**
Either:
1. You get back `USB_DRIVE_UNLOCKED` during the preflight checks to indicate to the client that it doesn't need to request the user's passphrase, then the client provides the export without an `encryption_key` and the export is successful.
2. We just always ask the user for their passphrase and the export is successful

**Actual**
You get back `USB_BAD_PASSPHRASE` and the export is not successful

**Comment by creviera** [comment]
Sounds like this has been repro'd but keeping label so in case someone wants to repro this less-than-ideal fix:

Unplug usb device, plug back in, try again. I don't think this will work if you say to remember the passphrase forever though.

**Comment by redshiftzero** [comment]
I tried to reproduce this before attempting a fix but it seems like this is resolved on `master`. To repro I unlocked my export drive manually in `sd-export-usb` prior to clicking export in the client. The export still occurred successfully. 

Looking at the code, this if statement should handle the case where the drive has already been unlocked: https://github.com/freedomofpress/securedrop-export/blob/master/securedrop_export/export.py#L247

If you agree @creviera, I think we can close this one
